pkgs/development/libraries/libnest2d/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, cmake, clipper, nlopt, boost, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "4.8.0";
```

Comment:
libnest2d is a third-party package that has been (hopefully temporarily) forked by Ultimaker, which explains why they didn't tag a release. Ultimaker's changes have been submitted upstream, but are not yet merged: https://github.com/tamasmeszaros/libnest2d/pull/18.

I think the version should be changed to `2020-10-09-unstable`, and comment should be added to the source saying we should switch to upstream if a release is made that contains the PR linked above.

Reply:
Done!